src/fetch/contracts.py

```diff
@@ -107,6 +110,40 @@ def contracts(self, load_from: NetworkFile) -> dict[str, bool]:
         write_to_csv(data_list=confirmed_contracts, outfile=load_file)
         return results
 
+    def get_non_wallets(self, contracts: list[str]):
+        code_at = self.batch_call(
+            addresses=contracts,
+            func=self._get_code_at
+        )
+        collector = defaultdict(list)
+        for account, code in code_at.items():
+            collector[code].append(account)
+
+        total_wallets = 0
+        wallets = set()
+        not_wallets = {}
+        unverified = {}
+        for byte_code, accounts in collector.items():
+            if byte_code in WALLET_BYTECODE:
+                wallets |= set(accounts)
+                total_wallets += len(accounts)
+            elif byte_code in NOT_WALLET_BYTECODE:
+                not_wallets[NOT_WALLET_BYTECODE[byte_code]] = accounts
```

Comment:
Seems like this case can be dropped entirely. If you want to keep it to not mess with the assertion in the `else` case a `pass` would be fine here, right?
In that case `not_wallets` can also be dropped.

Reply:
I think the idea was that if someone wanted more verbose logging about what's what, they could use these and add print statements. However, in its current state I completely agree it can be removed. I'm a bit hesitant though because I like the idea of seeing more details here. What do you think?

Reply:
I think optimizing the code for readability makes more sense than optimizing for people **maybe** needing more info. It's not difficult to add log info when you are debugging something.